num_2_item_dict: {"1" :"i_am_a_bunny_book",
"2" :"laugh_out_loud_joke_book",
"3" :"scotch_bubble_mailer",
"4" :"up_glucose_bottle",
"5" :"dasani_water_bottle",
"6" :"rawlings_baseball",
"7" :"folgers_classic_roast_coffee",
"8" :"elmers_washable_no_run_school_glue",
"9" :"hanes_tube_socks",
"10":"womens_knit_gloves",
"11": "cherokee_easy_tee_shirt",
"12": "peva_shower_curtain_liner",
"13": "cloud_b_plush_bear",
"14": "barkely_hide_bones",
"15": "kyjen_squeakin_eggs_plush_puppies",
"16": "cool_shot_glue_sticks",
"17": "creativity_chenille_stems",
"18": "soft_white_lightbulb",
"19": "safety_first_outlet_plugs",
"20": "oral_b_toothbrush",
"21": "dr_browns_bottle_brush",
"22": "command_hooks",
"23": "easter_turtle_sippy_cup",
"24": "fiskars_scissors_red",
"25": "scotch_duct_tape",
"26": "woods_extension_cord",
"27": "platinum_pets_dog_bowl",
"28": "fitness_gear_3lb_dumbbell",
"29": "rolodex_jumbo_pencil_cup",
"30": "clorox_utility_brush",
"31": "kleenex_paper_towels",
"32": "expo_dry_erase_board_eraser",
"33": "kleenex_tissue_box",
"34": "ticonderoga_12_pencils",
"35": "crayola_24_ct",
"36": "jane_eyre_dvd",
"37": "dove_beauty_bar",
"38": "staples_index_cards"}

item_2_num_dict: {"i_am_a_bunny_book":"1",
"laugh_out_loud_joke_book":"2",
"scotch_bubble_mailer":"3",
"up_glucose_bottle":"4",
"dasani_water_bottle":"5",
"rawlings_baseball":"6",
"folgers_classic_roast_coffee":"7",
"elmers_washable_no_run_school_glue":"8",
"hanes_tube_socks":"9",
"womens_knit_gloves":"10",
"cherokee_easy_tee_shirt":"11",
"peva_shower_curtain_liner":"12",
"cloud_b_plush_bear":"13",
"barkely_hide_bones":"14",
"kyjen_squeakin_eggs_plush_puppies":"15",
"cool_shot_glue_sticks":"16",
"creativity_chenille_stems":"17",
"soft_white_lightbulb":"18",
"safety_first_outlet_plugs":"19",
"oral_b_toothbrush":"20",
"dr_browns_bottle_brush":"21",
"command_hooks":"22",
"easter_turtle_sippy_cup":"23",
"fiskars_scissors_red":"24",
"scotch_duct_tape":"25",
"woods_extension_cord":"26",
"platinum_pets_dog_bowl":"27",
"fitness_gear_3lb_dumbbell":"28",
"rolodex_jumbo_pencil_cup":"29",
"clorox_utility_brush":"30",
"kleenex_paper_towels":"31",
"expo_dry_erase_board_eraser":"32",
"kleenex_tissue_box":"33",
"ticonderoga_12_pencils":"34",
"crayola_24_ct":"35",
"jane_eyre_dvd":"36",
"dove_beauty_bar":"37",
"staples_index_cards":"38"}

bin_2_num_dict: { "bin_A": 0,
                  "bin_B": 1,
                  "bin_C": 2,
                  "bin_D": 3,
                  "bin_E": 4,
                  "bin_F": 5,
                  "bin_G": 6,
                  "bin_H": 7,
                  "bin_I": 8,
                  "bin_J": 9,
                  "bin_K": 10,
                  "bin_L": 11}

current_bin_name: "bin_A"
current_item_number: 1

item_2_score_dict: {"1" : 0.125, #i_am_a_bunny_book,
"2" : 0.80, #laugh_out_loud_joke_book,
"3" : 0.90, #scotch_bubble_mailer,
"4" : 0.60, #up_glucose_bottle,
"5" : 0.40, #dasani_water_bottle,
"6" : 0.89, #rawlings_baseball,
"7" : 0.90, #folgers_classic_roast_coffee,
"8" : 0.32, #elmers_washable_no_run_school_glue,
"9" : 0.78, #hanes_tube_socks,
"10": 0.44, #womens_knit_gloves,
"11": 0.60, # cherokee_easy_tee_shirt,
"12": 0.60, # peva_shower_curtain_liner,
"13": 0.60, # cloud_b_plush_bear,
"14": 0.80, # barkely_hide_bones,
"15": 0.89, # kyjen_squeakin_eggs_plush_puppies,
"16": 0.33, # cool_shot_glue_sticks,
"17": 0.60, # creativity_chenille_stems,
"18": 0.67, # soft_white_lightbulb,
"19": 0.86, # safety_first_outlet_plugs,
"20": 0.10, # oral_b_toothbrush_green,
"21": 0.55, # dr_browns_bottle_brush,
"22": 0.15, # command_hooks,
"23": 0.67, # easter_turtle_sippy_cup,
"24": 0.57, # fiskars_scissors_red,
"25": 0.50, # scotch_duct_tape,
"26": 0.22, # woods_extension_cord,
"27": 0.70, # platinum_pets_dog_bowl,
"28": -1.0, # fitness_gear_3lb_dumbbell,
"29": 0.60, # rolodex_jumbo_pencil_cup,
"30": 0.44, # clorox_utility_brush,
"31": 1.00, # kleenex_paper_towels,
"32": 0.46, # expo_dry_erase_board_eraser,
"33": 0.90, # kleenex_tissue_box,
"34": 0.60, # ticonderoga_12_pencils,
"35": 0.67, # crayola_24_ct,
"36": 0.30, # jane_eyre_dvd,
"37": 0.88, # dove_beauty_bar,
"38": 0.53, # staples_index_cards}
"39": 0.10} #oral_b_toothbrush_red"

n_2_score_dict: {"1": 1.0,
"2": 1.0,
"3": 1.0,
"4": 0.6,
"5": 0.6,
"6": 0.6,
"7": 0.5,
"8": 0.35,
"9": 0.35,
"10": 0.35 }